src/storage/v2/storage.cpp

```diff
@@ -1980,6 +1980,11 @@ utils::BasicResult<Storage::CreateSnapshotError> Storage::CreateSnapshot(std::op
   return CreateSnapshotError::ReachedMaxNumTries;
 }
 
+bool Storage::IsPathLocked() {
+  auto locker_accessor = global_locker_.Access();
+  return locker_accessor.IsPathLocked(config_.durability.storage_directory);
+}
```

Comment:
For isPathLocked, I am not sure, but maybe we can do similar, and return enum, PathLocked::True, PathLocked::False, PathLocked::NonExistingPath? What do you think?

Reply:
I would like to have all 3 calls have the same behavior. Could we just use the BasicResult from the LockPath?
Or is that more for error passing?

Reply:
As we talked about this, I will resolve comment